server/src-lib/Hasura/GraphQL/Transport/HTTP.hs

```diff
@@ -120,22 +122,22 @@ runHasuraGQ
      , MonadQueryLog m
      )
   => RequestId
-  -> GQLReqUnparsed
+  -> (GQLReqUnparsed, GQLReqParsed)
   -> UserInfo
-  -> E.ExecOp
+  -> E.ExecOp (LazyTx QErr)
   -> m (DiffTime, Telem.QueryType, HTTP.ResponseHeaders, EncJSON)
   -- ^ Also return 'Mutation' when the operation was a mutation, and the time
   -- spent in the PG query; for telemetry.
-runHasuraGQ reqId query userInfo resolvedOp = do
+runHasuraGQ reqId (query, _queryParsed) userInfo resolvedOp = do
   (E.ExecutionCtx logger _ pgExecCtx _ _ _ _ _) <- ask
-  logQuery' logger
-  (telemTimeIO, respE) <- withElapsedTime $ liftIO $ runExceptT $ case resolvedOp of
-    E.ExOpQuery tx _genSql -> do
+  (telemTimeIO, respE) <- withElapsedTime $ runExceptT $ case resolvedOp of
+    E.ExOpQuery tx genSql _asts -> do
       -- log the generated SQL and the graphql query
-      -- L.unLogger logger $ QueryLog query genSql reqId
-      ([],) <$> runQueryTx pgExecCtx tx
+      logQueryLog logger query genSql reqId
```

Comment:
@sordina Do you have any idea on what these changes are related to?

Reply:
This is probably a leaked bit of the tracing implementation.